### PR TITLE
example: img-fifo script

### PR DESCRIPTION
**Pull request by justchokingaround**
i was messing around with ueberzugpp and fifo, and came up with this proof of concept code that allows a user to display an image in their terminal in the (probably) easiest way possible, using fifo. i also wrote a short script to achieve the same thing as `fzfub` does, but using fifo, please let me know if you'd like me to make a pull request for it

### CODE REVIEW

File: scripts/img [hunk 1]
```diff
@@ -23,4 +23,4 @@ ueberzugpp layer --no-stdin --silent --use-escape-codes --pid-file "$UB_PID_FILE
 UB_PID="$(cat "$UB_PID_FILE")"
 export UB_SOCKET=/tmp/ueberzugpp-"$UB_PID".socket
 ueberzugpp cmd -s "$UB_SOCKET" -a add -i PREVIEW -x "$1" -y "$2" --max-width "$3" --max-height "$4" -f "$5"
-read -r
+sleep 2
```
Comment (jstkdng) [inline scripts/img:26]: Why this change?

Comment (justchokingaround) [reply scripts/img:26]: idk how this happened, but before i pushed my code the first time, during testing, `read -r` worked fine, but now it doesn't work anymore, ueberzugpp just exits immediately. idk what caused this, sorry

**Comment by jstkdng** [comment]
Yeah sure, add the other fzf script, but I'd say add all to a fifo directory.

File: scripts/img (hunk 1 above)
Comment (jstkdng) [reply scripts/img:26]: Did that happen when you first uploaded the script?

**Comment by justchokingaround** [comment]
> Yeah sure, add the other fzf script, but I'd say add all to a fifo directory.

no, it did not

**Comment by justchokingaround** [comment]
> Yeah sure, add the other fzf script, but I'd say add all to a fifo directory.

yep, i will do that np

**Pull request merged**